charts/cloudzero-agent/README.md

```diff
@@ -163,7 +163,7 @@ See the `kube-state-metrics` [documentation](https://github.com/kubernetes/kube-
 | host              | string | `"api.cloudzero.com"` | CloudZero host to send metrics to.                                                                                      |
 | apiKey            | string | `nil`                 | The CloudZero API key to use to export metrics. Only used if `existingSecretName` is not set.                           |
 | existingSecretName| string | `nil`                 | The name of the secret that contains the CloudZero API key. Required if not providing the API key via the apiKey value. |
-| region            | string | `nil`                 | Region the cluster is running in.                                                                                       |
+| region            | string | `nil`                 | Region the cluster is running in. For example: `us-east-1`, `eastus`, `us-central1`, etc.      |
```

Comment:
it might be nice to link to the docs for the available cloud regions for each cloud provider just to make extra clear what we expect